docs/api/remote-procedure-calls-wallet.md

```diff
@@ -434,15 +434,13 @@ _See also_
 
 ## GetAddressInfo
 
-> 👍
->
-> Added in Dash Core 0.17.0
+**Added in Dash Core 0.17.0**
 
 > 📘
 >
 > Requires [wallet](../resources/glossary.md#wallet) support (**unavailable on masternodes**).
 
-The [`getaddressinfo` RPC](../api/remote-procedure-calls-wallet.md#getaddressinfo) returns information about the given Dash address. Note: Some information requires the address to be in the wallet.
+The [`getaddressinfo` RPC](../api/remote-procedure-calls-wallet.md#getaddressinfo) `labels` field now returns an array of label name strings. Previously, it returned an array of JSON objects containing `name` and `purpose` key/value pairs, which is now deprecated and will be removed in 0.21. To re-enable the previous behavior, launch bitcoind with `-deprecatedrpc=labelspurpose`.
```

Comment:
Let's leave the description here as-is and move the info about the `labels` field into the description for that field in the Result section below. That labels field also is simplified to now only include "Label name" strings instead of all the stuff there now.
I.e. that field will contain stuff like:
```
  "labels": [
    "My label"
  ]
```